app.rb

```diff
@@ -0,0 +1,107 @@
+require './person'
+require './book'
+require './classroom'
+require './rental'
+require './student'
+require './teacher'
+#rubocop:disable all
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def all_books
+    @books.each do |book|
+      puts "Title: #{book.title} , Author: #{book.author}"
+    end
+  end
+
+  def all_people
+    @people.each do |person|
+      puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+  end
+
+  def create_person
+    puts 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    person_type = gets.chomp
+
+    if person_type != '1' && person_type != '2'
+      puts 'Invalid option'
+      nil
+    end
+
+    print 'Age:'
+    age = gets.chomp
+
+    print 'Name:'
+    name = gets.chomp
+
+    person =
+      case person_type
+      when '1'
+        print 'Has parent permission? [Y/N]: '
+        parent_permission = gets.chomp
+        parent_permission = parent_permission.downcase == 'y'
+
+        Student.new(age, name, parent_permission)
+      when '2'
+        print 'Specialization:'
+        specialization = gets.chomp
+
+        Teacher.new(age, specialization, name)
+      end
+
+    @people << person
+    puts 'Person created'
+  end
+
+  # *Create a book.
+  def create_book
+    print 'Title: '
+
+    title = gets.chomp
+
+    print 'Author: '
+
+    author = gets.chomp
+
+    @books << Book.new(title, author)
+    puts 'Book created'
+  end
+
+  def create_rental
+    puts 'Select a book from the following list by number'
+    @books.each_with_index do |book, index|
+      puts "#{index} Title: #{book.title}, Author: #{book.author}"
+    end
+
+    book_rental = gets.chomp.to_i
+
+    print "\nSelect a person from th following list by number (not id)"
+    @people.each_with_index do |person, index|
+      puts "#{index} #{person.class} Name: #{person.name}, Id: #{person.id}, Age: #{person.age}"
```

Comment:
- [ ] Kindly ensure that a registered person's detail fall on the next line, as can be seen from the [video example](https://www.youtube.com/watch?v=vkkgrhD6aXQ)

![image](https://user-images.githubusercontent.com/12745474/142913764-cbcb6585-7c4c-404c-9505-f6f64c8b67ec.png)

Reply:
This issue has been addressed well. Good job! :clap: :clap: